dscript/language_model.py

```diff
@@ -94,7 +94,7 @@ def embed_from_fasta(fastaPath, outputPath, device=0, verbose=False):
             )
         )
 
-    h5fi = h5py.File(outputPath, "w")
+    h5fi = h5py.File(outputPath, "a")
```

Comment:
This is a minor nit, but I would suggest putting this in a context block using `with`.  I don't think the general users will care, but it'll clean up the need for the `h5fi.close()` calls.